src/explore-education-statistics-common/src/modules/table-tool/utils/mapColumns.ts

```diff
@@ -0,0 +1,45 @@
+import { ExpandedHeader, TableCellJson } from './mapTableToJson';
+
+export default function mapColumns(
```

Comment:
note LH: `mapColumns` and `mapRows` have been renamed to `mapTableBody` and `mapTableHead` respsectively

Reply:
Done, moved into `mapTableJson` as a private function. I think a big reason a lot of these functions got pulled out into their own file was due to not knowing exactly where they should go / just trying to make things as simple as possible in order to understand `mapTableToJson` a bit more clearly. I think once I add some more detailed comments to each function, it should become clear as to why they're all encapsulated in the `mapTableToJson` function instead of their own file